chains/v3/chains_dev.json

```diff
@@ -1867,7 +1871,12 @@
                 "symbol": "GLMR",
                 "precision": 18,
                 "priceId": "moonbeam",
-                "staking": "parachain"
+                "staking": "parachain",
+                "buyProviders": {
+                    "transak": {
+                       "network": "MOONRIVER"
```

Comment:
Most likely it should be "MOONBEAM" or "MAINNET", I'm not sure

Reply:
in addition to that, make sure to fix that in chains.json as well

Reply:
I bet it should be "MAINNET" according to:
https://www.notion.so/d20db13dcacb4ec8801e8869e2e5975e?v=9bcdb10756834041811b64d527b5fdc1